### PR TITLE
Eol content language check

### PR DESCRIPTION
**Pull request by sughics**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -144,6 +144,10 @@ dependencies {
 
     compile "com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:20180219.1"
 
+    implementation 'org.apache.tika:tika-core:2.4.1'
+    implementation 'org.apache.tika:tika-langdetect:2.4.1'
+    implementation 'org.apache.tika:tika-langdetect-optimaize:2.4.1'
+
```
Comment (charvolant) [inline build.gradle:150]: Is it possible to reduce the number of imports via dependencies?

File: grails-app/services/au/org/ala/bie/ExternalSiteService.groovy [hunk 2]
```diff
@@ -182,6 +186,22 @@ class ExternalSiteService implements GrailsConfigurationAware {
...
+     * @param language
+     * @param title
+     * @return boolean
+     */
+    Boolean isContentInLanguage(String content, String language, String title){
+        LanguageDetector detector = new OptimaizeLangDetector().loadModels()
+        // remove any html tags to prevent mis-detection
+        def cleanedContent  = Jsoup.parse(content).text()
+        LanguageResult result = detector.detect(cleanedContent)
+        log.warn("Content Language Detection for " + title)
```
Comment (charvolant) [inline grails-app/services/au/org/ala/bie/ExternalSiteService.groovy:201]: This is going to generate two log messages for every request. Perhaps just warn if there is a mismatch?

**Review by charvolant — CHANGES_REQUESTED** [review]
Could we make the language detecting a little less "loggy" please?

File: grails-app/services/au/org/ala/bie/ExternalSiteService.groovy (hunk 2 above)
Comment (sughics) [reply grails-app/services/au/org/ala/bie/ExternalSiteService.groovy]: Updated.

File: build.gradle (hunk 1 above)
Comment (sughics) [reply build.gradle:149]: Removed import for  `'org.apache.tika:tika-langdetect:2.4.1'` as it is accessible via tika-core.

**Review by charvolant — APPROVED** [review]

**Pull request merged**